Package.swift

```diff
@@ -5,6 +5,9 @@ import PackageDescription
 
 let package = Package(
     name: "Router",
+    platforms: [
+        .iOS(.v11)
```

Comment:
macOS, too

Reply:
Not yet right? 😛

Reply:
Nope, it doesn't build currently on macOS